test/annexB/language/function-code/block-decl-func-skip-arguments.js

```diff
@@ -0,0 +1,54 @@
+/*---
```

Comment:
We're missing license information here. I'd add it myself, but I'm not sure how to attribute it. @littledan would you mind submitting a patch with the appropriate text?